package.js

```diff
@@ -1,14 +1,14 @@
 Package.describe({
   name: 'meteoric:ionic-sass',
-  summary: "Ionic's CSS Framework in SASS and bundled for Meteor.",
-  version: '0.1.9',
+  summary: 'Ionic\'s CSS Framework in SASS and bundled for Meteor.',
+  version: '0.2.0',
   git: 'https://github.com/meteoric/ionic-sass'
 });
 
 Package.onUse(function(api) {
   api.versionsFrom('1.0');
-  api.use("fourseven:scss@2.0.0", ["server"]);
-  api.imply("fourseven:scss", ["server"]);
+  api.use('fourseven:scss@3.2.0', ['server']);
```

Comment:
@gwendall this is a breaking change that shouldn't have been snuck in with this PR. It appears `fourseven:scss` has changed quite a bit and now requires a `scss.json` file? Can you share how you are using this?